website/pages/docs/plugins/destinations/gremlin/overview.mdx

```diff
@@ -78,3 +78,7 @@ This is the (nested) spec used by the Gremlin destination Plugin.
 - `max_concurrent_connections` (integer, optional. default: number of runtime processors)
 
   Maximum number of concurrent connections to the database.
+
+- `neptune_compatibility` (boolean, optional. default: true)
```

Comment:
Another option would be to have an option like `use_full_types` (which would default to `false`)

Reply:
`complete_types`?

Reply:
Good one, implemented in https://github.com/cloudquery/cloudquery/pull/10846/commits/b19f8c68ba639e9cbdf39c3056b187bb373166f8